### ISSUE TITLE
Precision on Date Time Utils

### ISSUE THREAD

**Issue by lfgcampos** [AxonFramework/extension-amqp#94]
### Basic information

* Axon Framework version: 4.5.1
* JDK version: any
* AMQP Extension version: master
* Complete executable reproducer if available (e.g. GitHub Repo): #84 

While dependabot tried to upgrade extension Axon Framework version from 4.5 to 4.5.1, some of the tests started to fail. This is due to the changes made on Axon Framework https://github.com/AxonFramework/AxonFramework/issues/1749. Tests should be revisited and fixed.

### Steps to reproduce

Rerun #84 

### Expected behaviour

All tests should pass.

### Actual behaviour

Tests are failing.

**Comment by ZLStas** [comment]
Is it valid issue now?, I see <axon.version>4.5.2</axon.version> in pom file, and all the tests are passing.

**Comment by lfgcampos** [comment]
That's correct, it was fixed on this commit https://github.com/AxonFramework/extension-amqp/commit/276331b70e9b12a5f41197fbaa0c611ed8541d24.

Thanks for the heads up @ZLStas.
